### ISSUE TITLE
Unusable currently...

### ISSUE THREAD

**Issue by iamkether** [adjohnson916/firedux#3]
`import Firedux from 'firedux'` gives me: `Uncaught Error: Cannot find module "firedux"`

which appears to be happening because your `package.json` says this: `"main": "dist/index.js",` which doesn't actually exist. So I tried pathing directly to it at `firedux/dist/src`, and I get returned an empty object: `firedux:  Object {}`

Thought maybe it was console voodoo going on so I try the next step:

```const firedux = new Firedux({
  url: 'https://redux-firebase.firebaseio.com/'
})```

and I'm getting this error: `Uncaught TypeError: _src2.default is not a function`

Let me know what else I can do to help you fix this as I would love to start playing with this library.

**Comment by adjohnson916** [comment]
@iamkether Thanks for your interest and for reporting this obvious issue! Apologies - I refactored my ES6 compilation process at one point and apparently forgot to update `main` in `package.json`. I'll have a fix here shortly and push it up to npm ASAP.

**Comment by adjohnson916** [comment]
@iamkether Try `v0.5.0` just published to npm. If npm's caches haven't updated yet, you can install directly for now via:
```
npm i --save adjohnson916/firedux#v0.5.0
```

**Comment by adjohnson916** [comment]
@iamkether FYI I just released `v0.6.0` which simplifies the API by not requiring the passing of the `dispatch` function to each Firedux function, instead requiring that it be configured once on the Firedux instance for global use. See updated README for new usage.

**Comment by iamkether** [comment]
Awesome, thanks a lot for your speedy response @adjohnson916! I'll let you know if I build anything fun with it :)

**Comment by adjohnson916** [comment]
@iamkether FYI, I've published version `0.6.2` to address a similar error reported in #6, in case you ran into that.
